Kernel/Devices/KCOVInstance.h

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright (c) 2021, Patrick Meyer <git@the-space.agency>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#pragma once
+
+#include <Kernel/SpinLock.h>
+#include <Kernel/VM/AnonymousVMObject.h>
+#include <sys/kcov.h>
```

Comment:
Generally we avoid including userspace headers in the kernel. Types are either completely duplicated, or shoved into the grab-bag of UnixTypes.h. There's a few exceptions, but they seem to all reference the associated LibFoo in the include path (e.g. LibC/sys/ioctl_numbers.h)

Reply:
Looking at UnixTypes.h I don't think this stuff would fit in there very well. Would you recommend duplicating the two lines from kcov.h into KCOVInstance.h instead?

Reply:
Yeah duplication in Instance.h seems reasonable

Reply:
ack